dbcrossbarlib/src/drivers/postgres_shared/catalog.rs

```diff
@@ -200,6 +200,7 @@ fn pg_data_type(
         let ty = match data_type {
             "bigint" => Ok(PgScalarDataType::Bigint),
             "boolean" => Ok(PgScalarDataType::Boolean),
+            "character" => Ok(PgScalarDataType::Text),
```

Comment:
I love the implication here: "FFS, it's all just text"

Reply:
Yeah, basically we're choosing to live in denial unless it makes a difference, like it does for `json` and `jsonb`, which need separate handing in the PostgreSQL BINARY format.

`dbcrossbar` is one giant exercise in compromise and negotiation. It's the only way to get something like this off the ground gradually.